labs/src/main/scala/org/scalalabs/basic/lab02/ListManipulationExercise01.scala

```diff
@@ -88,7 +79,7 @@ object ListManipulationExercise01 {
    * pattern match or some other method.
    */
   def oddElements(iList: List[Int]): List[Int] = {
-    error("fix me")
+    iList.zipWithIndex.collect{ case (e, i) if ((i + 1) % 2) != 0 => e }
```

Comment:
I was pretty sure it wanted the odd-valued elements because of its suggestion of using filter. But yay we know how to do both